### ISSUE TITLE
Error for example in readme

### ISSUE THREAD

**Issue by FerusAndBeyond** [conradbez/hyperstream#6]
I get:

```
/hstream/components.py", line 179, in Components
    self, label: list[str], default_value: str = False, key: str = None
TypeError: 'type' object is not subscriptable
```

From running it.

**Comment by conradbez** [comment]
Is this fixed in the latest README / `hstream==0.0.15`?

**Comment by FerusAndBeyond** [comment]
It works, but now another error appears in its place, `ModuleNotFoundError: No module named 'websockets.datastructures`
